src/rust/src/x509/ocsp_req.rs

```diff
@@ -188,7 +189,8 @@ fn create_ocsp_request(
         (issuer_name_hash, issuer_key_hash, py_serial, py_hash) = builder
             .getattr(pyo3::intern!(py, "_request_hash"))?
             .extract()?;
-        let serial_number = asn1::BigInt::new(py_uint_to_big_endian_bytes(py, py_serial)?).unwrap();
+        let serial_number_bytes = ka.add(py_uint_to_big_endian_bytes(py, py_serial)?);
```

Comment:
I don't think a `ka` is needed here. Can just do the scoping trick.

Reply:
good catch, fixed